Makefile

```diff
@@ -41,7 +41,7 @@ migrate: ## Migrate the wagtail bakery site migrations
 	docker compose exec web python manage.py migrate
 
 test: ## Run all wagtail tests or pass in a file with `make test file=wagtail.admin.tests.test_name`
-	docker compose exec -w /code/wagtail web python runtests.py $(file) $(FILE)
+	docker compose exec -w /code/wagtail web python runtests.py $(file) $(FILE) --parallel
```

Comment:
We should probably add a separate command for this (or make it possible to pass additional arguments to `make test` to be passed on to `runtests.py`). Running tests in parallel is definitely much faster, but sometimes Django/unittest cannot produce a helpful traceback due to some pickling issues, in which case you'd still want to run it in serial.

Reply:
We could perhaps just add a separate command that runs in parallel for time being? E.g. have both `make test` and `make test-parallel` in the Makefile. @smark-1 could you change the PR so that we have these two commands?

Unless @laymonage has a goto solution for passing arguments from the `make` command, which to me seems like a slightly bigger task.

Reply:
@saevarom I updated the PR. I am not sure what the best way to document this is.  

1. `test-parallel` be a new subsection in the docs? 
2. Or should be mentioned as an comment above each `make test ...` option? 
3. Or a paragraph somewhere saying your can replace make test with make test-parallel to run faster but it sometimes cannot produce a helpful traceback?
4. other?